Copy edits to 2q and edb repositories and air-gapped topics (#115)
EnterpriseDB/tpa: docs/src/2q_and_edb_repositories.md

    - postgres_distributed
```

This example configures the `enterprise` and `postgres_distributed`

> repositories, giving access to EDB Postgres Advanced Server and PGD5 products.

Author: Is PGD5 a product separate from PGD? We don't refer to it as PGD5 if you mean PGD version 5.

Reviewer: it's referring to PGD version 5

Author: ```suggestion
repositories, giving access to EDB Postgres Advanced Server and PGD version 5 products.
```

> On Debian or Ubuntu systems, it uses the APT repository. 

Author: ```suggestion
On Debian or Ubuntu systems, it uses the apt repository. 
```

    - products/bdr3/release
```

This example installs the pglogical3 and bdr3 release repositories.
On Debian and Ubuntu systems, it uses the APT repository, and on

> RedHat systems, it uses the YUM repository.

Author: Should YUM be capitalized in this case? It isn't in the earlier cases but I wasn't sure if the repository name should be capped.

Reviewer: both yum and apt should be treated the same I don't think it needs to be capped, yum repository might even be replaced by rpm repository

Author: ```suggestion
RedHat systems, it uses the yum repository.
```
